layers/meta-resin-qemu/recipes-support/resin-runqemu/resin-runqemu.bb

```diff
@@ -0,0 +1,65 @@
+DESCRIPTION = "Resin helpers for running runqemu"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://runqemu-resin \
```

Comment:
Runqemu is meant to run in a build environment where PATH points to scripts and the sysroot is available. In this case this is changed and adapted only for the tools we provide in the bundle. In this way the sysroot will be the bundle and PATH will include the location of the bundle so we can find all the scripts there.